subsys/bluetooth/host/l2cap.c

```diff
@@ -376,6 +382,10 @@ static struct net_buf *l2cap_create_le_sig_pdu(struct net_buf *buf,
 					       u16_t len)
 {
 	struct bt_l2cap_sig_hdr *hdr;
+	struct net_buf_pool *pool = NULL;
+
+	if (code == BT_L2CAP_DISCONN_REQ)
+		pool = &disc_pool;
 
 	/* Don't wait more than the minimum RTX timeout of 2 seconds */
 	buf = bt_l2cap_create_pdu_timeout(NULL, 0, K_SECONDS(2));
```

Comment:
Did you intend to pass `pool` here? The variable seems unused otherwise.

Reply:
Will fix it.